ruby/exercises/palindromo.rb

```diff
@@ -7,3 +7,18 @@
   # 'Madam'
 # Ejemplo invalido:
 # 'Hello'
+def is_palindromo?(phrase)
+  p "[#{phrase}] es palindromo?"
+  phrase = phrase.delete(' ').downcase
```

Comment:
Siempre se recomiendo crear una variable adicional para no alterar el valor del argumento recibido
```suggestion
  parsed_phrase = phrase.delete(' ').downcase
```